devtools/docker-compose.yml

```diff
@@ -65,3 +65,30 @@ services:
           - api:api.dev.caliopen.org
       ports:
         - "4000:4000"
+  # Broker
+  broker:
+      build: ../src/backend/protocols/go.smtp
+      links: 
+          - apiv1:apiv1.dev.caliopen.org
+          - cassandra:cassandra.dev.caliopen.org
+          - elasticsearch:es.dev.caliopen.org
+          
+      ports:
+        - "2525:2525"
+        - "9042"
+  # NATS
+  nats:
+      build: ../src/backend/interfaces/NATS
```

Comment:
may get directly the official nats images instead of building it if there is nothing particular to configure